docs/data/date-pickers/fields/DateRangeFieldExamples.tsx

```diff
@@ -35,19 +32,10 @@ const GridItem = ({
 const date1 = dayjs('2022-04-07T14:30:22');
 const date2 = dayjs('2022-04-12T18:25:14');
 
-export default function FieldExamples() {
+export default function DateRangeFieldExamples() {
   return (
     <LocalizationProvider dateAdapter={AdapterDayjs}>
       <Grid container spacing={4} width={550}>
-        <GridItem label="DateField">
-          <DateField defaultValue={date1} />
-        </GridItem>
-        <GridItem label="TimeField">
-          <TimeField defaultValue={date1} />
-        </GridItem>
-        <GridItem label="DateTimeField">
-          <DateTimeField defaultValue={date1} />
-        </GridItem>
         <GridItem label="SingleInputDateRangeField">
           <SingleInputDateRangeField defaultValue={[date1, date2]} />
```

Comment:
Offtopic: Did we think about how are we going to handle `localeText.start` and `localeText.end` application to this and `MultiInput` components?
Currently both of these are not using those fields. 🤔

Reply:
Yes that is something we must support in the future indeed.
It would require wrapping the component in `LocalizationProvider` to forward the `localeText` prop to allow the override.
And then use the locale text to set the label.

But I propose to do it after #6392 to avoid conflicts.